### PR TITLE
Remove no longer needed dependencies

### LINKED ISSUE #260: Investigate switching from child_process
Based on discussion on PR https://github.com/apache/daffodil-vscode/pull/251, it seems the `child_process` is not actively maintained and that the package is available to actually be acquired. This should be investigated to why it is still the recommend package to use or investigate a different node package to use as an alternative.

### LINKED ISSUE #428: Remove vestigial dependencies
A while ago some of the client machinery moved from this repo over to Ωedit, to include things like protocol buffer and GRPC dependencies, but their dependencies are still in this repo's package.json.  This task is to remove those vestigial dependencies.  This helps to reduce dependabot PRs as well.

### PR DESCRIPTION
**Pull request by Shanedell**
Remove no longer needed dependencies:

- child_process was removed as it is provided by nodejs when installed.
- @grpc/grpc-js was removed as it is no longer used directly in this repo.
- google-protobuf was removed as it is no longer used directly in this repo.

Closes #260
Closes #428

### CODE REVIEW

**Review by mbeckerle — APPROVED** [review]
+1

File: yarn.lock [hunk 1]
```diff
@@ -14,10 +14,10 @@
...
   integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
 
-"@grpc/grpc-js@^1.5.4", "@grpc/grpc-js@^1.7.1":
-  version "1.8.3"
-  resolved "https://registry.yarnpkg.com/@grpc/grpc-js/-/grpc-js-1.8.3.tgz#1eceeeda8d3af6a3cf9f1b74032bbc411af9f3eb"
-  integrity sha512-1oO40SPA9nyBEsSMg0Mq7aMxQMVH009NJLG4HeMPQ7kdDGWuC5fI6dmEjaEJzUbtmC1S2leLaDRMB4ByZ1cFew==
+"@grpc/grpc-js@^1.7.1":
+  version "1.8.7"
+  resolved "https://registry.yarnpkg.com/@grpc/grpc-js/-/grpc-js-1.8.7.tgz#2154fc0134462ad45f4134e8b54682a25ed05956"
+  integrity sha512-dRAWjRFN1Zy9mzPNLkFFIWT8T6C9euwluzCHZUKuhC+Bk3MayNPcpgDRyG+sg+n2sitEUySKxUynirVpu9ItKw==
```
Comment (scholarsmate) [inline yarn.lock:20]: Why is this still here?

File: yarn.lock [hunk 2]
```diff
@@ -1142,11 +1137,6 @@ glob@^7.0.6, glob@^7.1.3:
...
     path-is-absolute "^1.0.0"
 
-google-protobuf@3.20.1:
-  version "3.20.1"
-  resolved "https://registry.yarnpkg.com/google-protobuf/-/google-protobuf-3.20.1.tgz#1b255c2b59bcda7c399df46c65206aa3c7a0ce8b"
-  integrity sha512-XMf1+O32FjYIV3CYu6Tuh5PNbfNEU5Xu22X+Xkdb/DUexFlCzhvv7d5Iirm4AOwn8lv4al1YvIhzGrg2j9Zfzw==
-
 google-protobuf@^3.21.2:
   version "3.21.2"
   resolved "https://registry.yarnpkg.com/google-protobuf/-/google-protobuf-3.21.2.tgz#4580a2bea8bbb291ee579d1fefb14d6fa3070ea4"
```
Comment (scholarsmate) [inline yarn.lock:1142]: Why is this still here?

File: yarn.lock [hunk 3]
```diff
@@ -1798,9 +1788,9 @@ process-nextick-args@~2.0.0:
   integrity sha512-3ouUOpQhtgrbOa17J7+uxOTpITYWaGP7/AhoR3+A+/1e9skrzelGi/dXzEYyvbxubEF6Wn2ypscTKiKJFFn1ag==
 
 protobufjs@^7.0.0:
-  version "7.1.2"
-  resolved "https://registry.yarnpkg.com/protobufjs/-/protobufjs-7.1.2.tgz#a0cf6aeaf82f5625bffcf5a38b7cd2a7de05890c"
-  integrity sha512-4ZPTPkXCdel3+L81yw3dG6+Kq3umdWKh7Dc7GW/CpNk4SX3hK58iPCWeCyhVTDrbkNeKrYNZ7EojM5WDaEWTLQ==
+  version "7.2.0"
+  resolved "https://registry.yarnpkg.com/protobufjs/-/protobufjs-7.2.0.tgz#ca6b1ceb9a9efe21186ba96178089ec563011a5e"
+  integrity sha512-hYCqTDuII4iJ4stZqiuGCSU8xxWl5JeXYpwARGtn/tWcKCAro6h3WQz+xpsNbXW0UYqpmTQFEyFWO0G0Kjt64g==
```
Comment (scholarsmate) [inline yarn.lock:1793]: Why is this still here?

**Review by scholarsmate — CHANGES_REQUESTED** [review]
Changes to package.json look good, but the dependencies still appear in yarn.lock.  Is that due to transitive dependencies in Ωedit?

File: yarn.lock (hunk 1 above)
Comment (scholarsmate) [reply yarn.lock:20]: Transitive deps.

File: yarn.lock (hunk 2 above)
Comment (scholarsmate) [reply yarn.lock:1142]: Transitive deps.

File: yarn.lock (hunk 3 above)
Comment (scholarsmate) [reply yarn.lock:1793]: Transitive deps.

**Review by scholarsmate — APPROVED** [review]

**Pull request merged**